src/mon/MDSMonitor.cc

```diff
@@ -1508,6 +1508,9 @@ class FlagSetHandler : public FileSystemCommandHandler
     string flag_val;
     cmd_getval(g_ceph_context, cmdmap, "val", flag_val);
 
+    string confirm;
+    cmd_getval(g_ceph_context, cmdmap, "confirm", confirm);
```

Comment:
doc/cephfs/administration.rst has a reference to enable_multiple that could be updated for this